README.md

```diff
@@ -1,74 +1,57 @@
-## VisualRegressionTests
+# VisualRegressionTests.jl
 
-Easy regression testing for visual packages.  Automated tests compare similarity between a newly generated image
-and a reference image using the Images package.  While in interactive mode, the tests can optionally pop up a
+Easy regression testing for visual packages. Automated tests compare similarity between a newly generated image
+and a reference image using the Images package. While in interactive mode, the tests can optionally pop up a
 Gtk GUI window showing a side-by-side comparison of the test and reference image, and then optionally overwrite
-the reference image with the test image.  This allows for straightforward regression testing of image data, even
+the reference image with the test image. This allows for straightforward regression testing of image data, even
 when the "correct" images change over time.
 
-#### Author: Thomas Breloff (@tbreloff)
```

Comment:
Yes, I think so. Even after we have completely rewritten it? I just find this line unnecessary as many people have contributed to the current version, which deviates a lot from the original implementation.

Reply:
https://github.com/JuliaPlots/VisualRegressionTests.jl/graphs/contributors Well, 4 people have contributed except from Tom. You have the most contributions, and I have no problems, if you want to add your name somewhere in the README. I just don't feel comfortable with removing this line, because it was Tom's idea and work originally.

Reply:
I will add the line back again, sorry for the excessive clean up.